### PR TITLE
Issue Template: improve formatting and add helpful tips

### PR DESCRIPTION
**Pull request by cvan**
adapted from https://www.talater.com/open-source-templates/#/page/98

<hr>

the heading formatting (`##`) and the numbered steps were the main additions I was going for here. I'm open to suggestions to the other changes. let me know!

### CODE REVIEW

**Comment by larsbergstrom** [comment]
I like this better! @nancyhang Any other additions/changes?

**Comment by MortimerGoro** [comment]
I like this one too. I'd also add a category to attach log or stacktraces (e.g Insert your log/stack trace here)

**Comment by cvan** [comment]
:+1: thanks, everyone, for the (quick and useful) feedback. addressed and merged. let me know if there's anything else you'd like me to add.

File: issue_template [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+
+## Context
+<!--- [Optional] -->
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world! :) -->
+
+## Error Logs and Stack Traces
+<!--- [Optional] -->
+<!--- These are very useful for quickly identifying the causes for bug fixes. -->
+<!--- In Developer Mode, run `adb logcat | grep GeckoView` to capture potentially useful logs. -->
```
[the remark this answers is not in the document]
Comment (bluemarvin) [reply issue_template]: Just a capture of the logcat is probably fine. Better to have too much than not enough.

[the remark this answers is not in the document]
Comment (MortimerGoro) [reply issue_template]: yes, I agree, the whole logcat capture is fine

**Comment by MortimerGoro** [comment]
@cvan could you remove the grep from thelogcat so we can land this?

**Comment by cvan** [comment]
> @cvan could you remove the grep from the logcat command so we can land this?

done 👍

File: issue_template [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+
+## Context
+<!--- [Optional] -->
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world! :) -->
+
+## Error Logs and Stack Traces
+<!--- [Optional] -->
+<!--- These are very useful for quickly identifying the causes for bug fixes. -->
+<!--- In Developer Mode, run `adb logcat` (or `adb logcat | grep GeckoView`) to capture potentially useful logs. -->
```
Comment (bluemarvin) [inline issue_template]: I'm not sure that you will get anything from adb logcat | grep GeckoView just adb logcat is fine.

Comment (bluemarvin) [reply issue_template]: looks like you also need to rebase?

Comment (cvan) [reply issue_template]: ## `adb logcat`

```
08-09 19:12:01.226  1203  1203 I ControllerNotifications: Updating input notifications based on state ConnectedBatteryNormal
08-09 19:12:01.575  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=3,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1017MHz,Free=1013MB,PSM=0,PLS=0,Temp=28.7C/25.0C,TW=2.22ms,App=7.12ms
08-09 19:12:01.575  4641  5442 I UtilPoller: GPU Util 0.562094 / CPU Util 0.505155 (avg 0.467443)
08-09 19:12:01.773   554   910 W ServiceManager: Permission failure: com.oma.drm.permission.ACCESS_OMA_DRM from uid=10136 pid=4641
08-09 19:12:01.777  4641  4660 D GeckoViewModule: dispatch GeckoView:Reload, data=null
08-09 19:12:01.786   554   910 W ServiceManager: Permission failure: com.oma.drm.permission.ACCESS_OMA_DRM from uid=10136 pid=4641
08-09 19:12:01.788   554   554 W ServiceManager: Permission failure: com.oma.drm.permission.ACCESS_OMA_DRM from uid=10136 pid=4641
08-09 19:12:01.789  4641  4660 D GeckoViewNavigation: onEvent: event=GeckoView:Reload, data=null
08-09 19:12:01.790   559   905 I ExtendedExtractor: QTIParser is prefered .. flags 0
08-09 19:12:01.790   559   905 D MMParserExtractor: setExtraFlags called with flags 0 for paser instance f55adc80
08-09 19:12:01.790   559   905 I ExtendedExtractor: ExtendedExtractor::create 0xf55adc80
08-09 19:12:01.798   565  5120 I MediaPlayerService: MediaPlayerService::getOMX
08-09 19:12:01.801  4641  5444 I OMXClient: MuxOMX ctor
08-09 19:12:01.803   557   897 I OMXMaster: makeComponentInstance(OMX.google.raw.decoder) in mediacodec process
08-09 19:12:01.805  4641  4660 D GeckoViewNavigation[C]: loadURI: uri=http://10.0.1.177:3000/wire-e where=1 flags=0
08-09 19:12:01.863  4641  4660 D GeckoViewProgress[C]: onStateChange: isTopLevel=true, flags=0xf0001, status=NS_OK
08-09 19:12:01.870  4641  4660 D GeckoViewProgress[C]: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:01.871  4641  4660 D GeckoViewProgress[C]: ProgressTracker start http://10.0.1.177:3000/wire-e
08-09 19:12:01.872  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:01.876  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":0,"uri":"http://10.0.1.177:3000/wire-e","locationChange":false,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":false,"parsed":false,"totalReceived":1,"totalExpected":1} 1/1 progress=10.000057220458984
08-09 19:12:01.878  4641  4660 D GeckoViewProgress: onStateChange: isTopLevel=true, flags=0xf0001, status=NS_OK
08-09 19:12:01.879  4641  4660 D GeckoViewProgress: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:01.895  4641  4641 W OpenGLRenderer: Points are too far apart 4.047089
08-09 19:12:01.895  4641  4641 W OpenGLRenderer: Points are too far apart 4.043456
08-09 19:12:02.503  4641  4660 D GeckoViewProgress[C]: onLocationChange: location=http://10.0.1.177:3000/wire-e, flags=0
08-09 19:12:02.504  4641  4660 D GeckoViewProgress[C]: ProgressTracker changeLocation http://10.0.1.177:3000/wire-e
08-09 19:12:02.505  4641  4660 D GeckoViewProgress: onLocationChange: location=http://10.0.1.177:3000/wire-e, flags=0
08-09 19:12:02.507  4641  4660 D GeckoViewNavigation: onLocationChange
08-09 19:12:02.509  4641  4660 D GeckoViewTrackingProtection: onSecurityChange
08-09 19:12:02.510  4641  4660 D GeckoViewProgress: onSecurityChange
08-09 19:12:02.514  4641  4660 D GeckoViewProgress[C]: onProgressChange 0/0 10083/10083
08-09 19:12:02.515  4641  4660 D GeckoViewProgress[C]: ProgressTracker handleProgress null 10083/10083
08-09 19:12:02.518  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.520  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":10.000057220458984,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":false,"parsed":false,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=20.577011108398438
08-09 19:12:02.531  4641  4660 D GeckoViewContent[C]: handleEvent: DOMTitleChanged
08-09 19:12:02.562  4641  4660 I GeckoConsole: No chrome package registered for chrome://browser/skin/forward.svg
08-09 19:12:02.578  4641  5442 I UtilPoller: GPU Util 0.581503 / CPU Util 0.616162 (avg 0.562133)
08-09 19:12:02.580  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1296MHz,Free=1013MB,PSM=0,PLS=0,Temp=28.7C/25.0C,TW=2.38ms,App=7.21ms
08-09 19:12:02.603  4641  4660 W GeckoConsole: [JavaScript Warning: "ReferenceError: ga is not defined"]
08-09 19:12:02.605  4641  4660 D GeckoViewProgress[C]: ProgressTracker handleEvent: DOMContentLoaded
08-09 19:12:02.606  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.607  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":20.577011108398438,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":false,"parsed":true,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=35.57701110839844
08-09 19:12:02.611  4641  4660 D GeckoViewProgress[C]: ProgressTracker handleEvent: pageshow
08-09 19:12:02.611  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.614  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":35.57701110839844,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":true,"parsed":true,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=50.57701110839844
08-09 19:12:02.625  4641  4660 D GeckoViewProgress[C]: onStateChange: isTopLevel=true, flags=0xc0010, status=NS_OK
08-09 19:12:02.626  4641  4660 D GeckoViewProgress[C]: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:02.626  4641  4660 D GeckoViewProgress[C]: ProgressTracker stop
08-09 19:12:02.627  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.629  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":50.57701110839844,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":true,"firstPaint":false,"pageShow":true,"parsed":true,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=100
08-09 19:12:02.630  4641  4660 D GeckoViewProgress: onStateChange: isTopLevel=true, flags=0xc0010, status=NS_OK
08-09 19:12:02.631  4641  4660 D GeckoViewProgress: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:03.576  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1017MHz,Free=931MB,PSM=0,PLS=0,Temp=28.7C/25.0C,TW=2.36ms,App=7.06ms
08-09 19:12:03.580  4641  5442 I UtilPoller: GPU Util 0.558961 / CPU Util 0.699029 (avg 0.610597)
08-09 19:12:04.575  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1296MHz,Free=957MB,PSM=0,PLS=0,Temp=28.7C/25.0C,TW=2.39ms,App=7.10ms
08-09 19:12:04.582  4641  5442 I UtilPoller: GPU Util 0.568266 / CPU Util 0.484848 (avg 0.447663)
08-09 19:12:05.575  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1017MHz,Free=913MB,PSM=0,PLS=0,Temp=28.7C/25.0C,TW=2.52ms,App=7.55ms
08-09 19:12:05.586  4641  5442 I UtilPoller: GPU Util 0.588705 / CPU Util 0.574257 (avg 0.541742)
08-09 19:12:06.573  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1017MHz,Free=914MB,PSM=0,PLS=0,Temp=28.7C/25.0C,TW=2.31ms,App=7.40ms
08-09 19:12:06.589  4641  5442 I UtilPoller: GPU Util 0.590572 / CPU Util 0.730000 (avg 0.670334)
08-09 19:12:07.573  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1017MHz,Free=910MB,PSM=0,PLS=0,Temp=29.0C/25.0C,TW=2.08ms,App=6.91ms
08-09 19:12:07.591  4641  5442 I UtilPoller: GPU Util 0.579086 / CPU Util 0.615385 (avg 0.600153)
08-09 19:12:08.302   554   554 W ServiceManager: Permission failure: com.oma.drm.permission.ACCESS_OMA_DRM from uid=10136 pid=4641
08-09 19:12:08.308   554  2271 W ServiceManager: Permission failure: com.oma.drm.permission.ACCESS_OMA_DRM from uid=10136 pid=4641
08-09 19:12:08.310   554  2271 W ServiceManager: Permission failure: com.oma.drm.permission.ACCESS_OMA_DRM from uid=10136 pid=4641
08-09 19:12:08.311   559   905 I ExtendedExtractor: QTIParser is prefered .. flags 0
08-09 19:12:08.311   559   905 D MMParserExtractor: setExtraFlags called with flags 0 for paser instance f3d0c280
08-09 19:12:08.311   559   905 I ExtendedExtractor: ExtendedExtractor::create 0xf3d0c280
08-09 19:12:08.317   565  1429 I MediaPlayerService: MediaPlayerService::getOMX
08-09 19:12:08.319  4641  5449 I OMXClient: MuxOMX ctor
08-09 19:12:08.320   557  1142 I OMXMaster: makeComponentInstance(OMX.google.raw.decoder) in mediacodec process
08-09 19:12:08.459   982  1046 D WifiStateMachine: updateCapabilities for config:⌥⌘⎋false,false
08-09 19:12:08.461  1554  1554 D com.oculus.telemetry.collector.WiFiCollector: action = android.net.wifi.RSSI_CHANGED
08-09 19:12:08.575  4641  5441 I VrApi   : FPS=60,Prd=31ms,Tear=0,Early=0,Stale=0,VSnc=1,Lat=0,Fov=0,CPU2/GPU=4/4,1593/510MHz,OC=F,TA=E/C/C,SP=N/N/N,Mem=1017MHz,Free=906MB,PSM=0,PLS=0,Temp=29.0C/25.0C,TW=2.44ms,App=7.24ms
08-09 19:12:08.593  4641  5442 I UtilPoller: GPU Util 0.579992 / CPU Util 0.640000 (avg 0.602014)
08-09 19:12:08.771   579   579 E cnss-daemon: Stale or unreachable neighbors, ndm state: 4
```

## `adb logcat | grep GeckoView`

```
08-09 19:12:01.777  4641  4660 D GeckoViewModule: dispatch GeckoView:Reload, data=null
08-09 19:12:01.789  4641  4660 D GeckoViewNavigation: onEvent: event=GeckoView:Reload, data=null
08-09 19:12:01.805  4641  4660 D GeckoViewNavigation[C]: loadURI: uri=http://10.0.1.177:3000/wire-e where=1 flags=0
08-09 19:12:01.863  4641  4660 D GeckoViewProgress[C]: onStateChange: isTopLevel=true, flags=0xf0001, status=NS_OK
08-09 19:12:01.870  4641  4660 D GeckoViewProgress[C]: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:01.871  4641  4660 D GeckoViewProgress[C]: ProgressTracker start http://10.0.1.177:3000/wire-e
08-09 19:12:01.872  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:01.876  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":0,"uri":"http://10.0.1.177:3000/wire-e","locationChange":false,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":false,"parsed":false,"totalReceived":1,"totalExpected":1} 1/1 progress=10.000057220458984
08-09 19:12:01.878  4641  4660 D GeckoViewProgress: onStateChange: isTopLevel=true, flags=0xf0001, status=NS_OK
08-09 19:12:01.879  4641  4660 D GeckoViewProgress: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:02.503  4641  4660 D GeckoViewProgress[C]: onLocationChange: location=http://10.0.1.177:3000/wire-e, flags=0
08-09 19:12:02.504  4641  4660 D GeckoViewProgress[C]: ProgressTracker changeLocation http://10.0.1.177:3000/wire-e
08-09 19:12:02.505  4641  4660 D GeckoViewProgress: onLocationChange: location=http://10.0.1.177:3000/wire-e, flags=0
08-09 19:12:02.507  4641  4660 D GeckoViewNavigation: onLocationChange
08-09 19:12:02.509  4641  4660 D GeckoViewTrackingProtection: onSecurityChange
08-09 19:12:02.510  4641  4660 D GeckoViewProgress: onSecurityChange
08-09 19:12:02.514  4641  4660 D GeckoViewProgress[C]: onProgressChange 0/0 10083/10083
08-09 19:12:02.515  4641  4660 D GeckoViewProgress[C]: ProgressTracker handleProgress null 10083/10083
08-09 19:12:02.518  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.520  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":10.000057220458984,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":false,"parsed":false,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=20.577011108398438
08-09 19:12:02.531  4641  4660 D GeckoViewContent[C]: handleEvent: DOMTitleChanged
08-09 19:12:02.605  4641  4660 D GeckoViewProgress[C]: ProgressTracker handleEvent: DOMContentLoaded
08-09 19:12:02.606  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.607  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":20.577011108398438,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":false,"parsed":true,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=35.57701110839844
08-09 19:12:02.611  4641  4660 D GeckoViewProgress[C]: ProgressTracker handleEvent: pageshow
08-09 19:12:02.611  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.614  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":35.57701110839844,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":false,"firstPaint":false,"pageShow":true,"parsed":true,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=50.57701110839844
08-09 19:12:02.625  4641  4660 D GeckoViewProgress[C]: onStateChange: isTopLevel=true, flags=0xc0010, status=NS_OK
08-09 19:12:02.626  4641  4660 D GeckoViewProgress[C]: onStateChange: uri=http://10.0.1.177:3000/wire-e
08-09 19:12:02.626  4641  4660 D GeckoViewProgress[C]: ProgressTracker stop
08-09 19:12:02.627  4641  4660 D GeckoViewProgress[C]: ProgressTracker updateProgress
08-09 19:12:02.629  4641  4660 D GeckoViewProgress[C]: ProgressTracker onProgressChangeUpdate {"prev":50.57701110839844,"uri":"http://10.0.1.177:3000/wire-e","locationChange":true,"pageStart":true,"pageStop":true,"firstPaint":false,"pageShow":true,"parsed":true,"totalReceived":10084,"totalExpected":10084} 10084/10084 progress=100
08-09 19:12:02.630  4641  4660 D GeckoViewProgress: onStateChange: isTopLevel=true, flags=0xc0010, status=NS_OK
08-09 19:12:02.631  4641  4660 D GeckoViewProgress: onStateChange: uri=http://10.0.1.177:3000/wire-e
```

seems to help filter a little. I'll remove the tip for the pipe command though.

Comment (bluemarvin) [reply issue_template]: It does filter, but the GeckoView logcat messages aren't the ones we need to see.

**Pull request merged**